### PR TITLE
Add redelegate command

### PR DESCRIPTION
**Pull request by joojis**

### CODE REVIEW

File: ccstake/src/cmds/redelegate.ts [hunk 1]
```diff
@@ -0,0 +1,175 @@
...
+    delegator: PlatformAddress;
+    "previous-delegatee": PlatformAddress;
+    "next-delegatee": PlatformAddress;
+    quantity: U64;
+    fee: number;
+}
+
+export const module: yargs.CommandModule<GlobalParams, RedelegateParams> = {
+    command: "redelegate",
+    describe: "Move a delegation from existing delegatee to another account",
```
Comment (joojis) [inline ccstake/src/cmds/redelegate.ts]: @ScarletBlue Please check here, too.

File: ccstake/src/cmds/redelegate.ts [hunk 2]
```diff
@@ -0,0 +1,175 @@
...
+        console.log("CCC Balance:", ...minusChangeArgs(cccBalance, cccChanges));
+        const { balance, undelegated, delegationsTo } = summary.get(delegator);
+        console.log("CCS Balance:", balance.toLocaleString());
+        console.log(
+            "Undelegated CCS:",
+            ...minusChangeArgs(undelegated, ccsChanges)
+        );
+        console.log(
+            "Delegations (out):",
+            ...plusChangeArgs(delegationsTo.sum, ccsChanges)
```
Comment (foriequal0) [inline ccstake/src/cmds/redelegate.ts]: Ditto.

File: ccstake/src/cmds/redelegate.ts [hunk 3]
```diff
@@ -0,0 +1,175 @@
...
+        const cccBalance = await sdk.rpc.chain.getBalance(
+            delegator,
+            blockNumber
+        );
+        console.log("CCC Balance:", ...minusChangeArgs(cccBalance, cccChanges));
+        const { balance, undelegated, delegationsTo } = summary.get(delegator);
+        console.log("CCS Balance:", balance.toLocaleString());
+        console.log(
+            "Undelegated CCS:",
+            ...minusChangeArgs(undelegated, ccsChanges)
```
Comment (foriequal0) [inline ccstake/src/cmds/redelegate.ts]: Redelegation doesn't change the delegator's total amount of CCS that are delegated to someone.

**Review by foriequal0 — CHANGES_REQUESTED** [review]

File: ccstake/src/cmds/redelegate.ts (hunk 2 above)
Comment (joojis) [reply ccstake/src/cmds/redelegate.ts]: Fixed it

File: ccstake/src/cmds/redelegate.ts (hunk 3 above)
Comment (joojis) [reply ccstake/src/cmds/redelegate.ts]: Fixed it

File: ccstake/src/cmds/redelegate.ts [hunk 4]
```diff
@@ -0,0 +1,169 @@
...
+    delegator: PlatformAddress;
+    "previous-delegatee": PlatformAddress;
+    "next-delegatee": PlatformAddress;
+    quantity: U64;
+    fee: number;
+}
+
+export const module: yargs.CommandModule<GlobalParams, RedelegateParams> = {
+    command: "redelegate",
+    describe: "Move a delegation from existing delegatee to another account",
```
Comment (ScarletBlue) [inline ccstake/src/cmds/redelegate.ts]:
```suggestion
    describe: "Move a delegation from an existing delegatee to another account",
```

**Review by ScarletBlue — CHANGES_REQUESTED** [review]

**Pull request merged**